File: c/01-estrutura-sequencial/troco.c
/*Problema "troco"
Fazer um programa para calcular o troco no processo de pagamento de um produto de uma mercearia.
O programa deve ler o preço unitário do produto, a quantidade de unidades compradas deste produto,
e o valor em dinheiro dado pelo cliente (suponha que haja dinheiro suficiente). Seu programa deve
mostrar o valor do troco a ser devolvido ao cliente.*/

#include <stdio.h> 
int main () {
    double price, moneyreceived, change; 
    int quantity;

    printf("PRECO UNITARIO: ");
    scanf("%lf", &price);
    printf("QUANTIDADE COMPRADA: ");
    scanf("%d", &quantity);
    printf("DINHEIRO RECEBIDO: ");
    scanf("%lf", &moneyreceived);

    change = moneyreceived - (price * quantity); 

    printf("TROCO = %.2lf\n", change); 
    
    return 0;
}

File: java/01-estrutura-sequencial/Troco.java
/*Problema "troco"
Fazer um programa para calcular o troco no processo de pagamento de um produto de uma mercearia.
O programa deve ler o preço unitário do produto, a quantidade de unidades compradas deste produto,
e o valor em dinheiro dado pelo cliente (suponha que haja dinheiro suficiente). Seu programa deve
mostrar o valor do troco a ser devolvido ao cliente.*/

import java.util.Locale;
import java.util.Scanner;

public class Troco {

	public static void main(String[] args) {
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		double price, moneyreceived, change;
		int quantity;
		
		System.out.print("Preço unitário do produto: ");
		price = sc.nextDouble();
		System.out.print("Dinheiro recebido: ");
		moneyreceived = sc.nextDouble();
		System.out.print("Quantidade: ");
		quantity = sc.nextInt();
		
		change = moneyreceived - (price * quantity); 
		
		System.out.println("Troco = " + String.format("%.2f", change));
		
		sc.close();
		
	}

}
